lte/gateway/c/session_manager/test/test_local_enforcer.cpp

```diff
@@ -828,6 +829,7 @@ TEST_F(LocalEnforcerTest, test_cwf_final_unit_handling) {
   test_cwf_cfg.common_context =
       build_common_context(IMSI1, "", "", "", "", TGPP_WLAN);
   const auto& wlan = build_wlan_context(MAC_ADDR, RADIUS_SESSION_ID);
+  test_cwf_cfg.common_context.set_rat_type(TGPP_WLAN);
```

Comment:
those tests were done on LTE by mistake
With this flag set to WLAN we force them to CWF